From cc6f7b09ed62061f5c170103478fa7161b3eb9df Mon Sep 17 00:00:00 2001
From: Matt Madison <matt@madison.systems>
Date: Mon, 27 Aug 2018 11:29:39 -0700
Subject: [PATCH] link: drop the forced use of gold on aarch64

as the original issue with BFD appears to have been fixed,
and using gold on aarch64 is causing other issues.
---
 src/cmd/link/internal/ld/lib.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cmd/link/internal/ld/lib.go b/src/cmd/link/internal/ld/lib.go
index 0b9e2d06db..e827fb1b54 100644
--- a/src/cmd/link/internal/ld/lib.go
+++ b/src/cmd/link/internal/ld/lib.go
@@ -1155,7 +1155,7 @@ func (l *Link) hostlink() {
 		// from the beginning of the section (like STYPE).
 		argv = append(argv, "-Wl,-znocopyreloc")
 
-		if SysArch.InFamily(sys.ARM, sys.ARM64) {
+		if SysArch.InFamily(sys.ARM) {
 			// On ARM, the GNU linker will generate COPY relocations
 			// even with -znocopyreloc set.
 			// https://sourceware.org/bugzilla/show_bug.cgi?id=19962
-- 
2.17.1

